index.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   
+    <header id="nav-bar">
+            <h1>
+                Shoaib
+            </h1>
+            <desktop id="header-desktop">
+            <h2 id="portfolio">Portoflio</h2>
+            <h3 id="about-header">About</h3>
+            <h4 id="contact-header">Contact</h4>
+            <img src="images/letter-icon.svg" id="letter-icon">
+            <img src="images/Union.png" id="menu">
+        </desktop>
+    </header>
+    <section id="home">
+        
+        <div id="intro">
+        <h1>
+            Hey There. I'm Shoaib
+        </h1>
+        <h2>
+            I’am a Software Developer
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+        </p>
+       
+       </div>
+       <div id="social-media">
+        <a href="#"><img src="images/github.svg" id="github"></a>
+        <a href="#"><img src="images/linkedin.svg" alt=""></a>
+        <a href="#"><img src="images/Hand.svg" alt=""></a>
+        <a href="#"><img src="images/twitter.svg" alt=""></a>
+        <a href="#"><img src="images/M.svg" alt=""></a>
+   
+       </div>
+       <img src="images/illustration-header-mobile.svg" id="home-id">
+       <img src="images/desktop-main.svg" id="desktop-main">
```

Comment:
- [ ] Good job on styling the image as a background image in HTML, but it's better to add the background image instead of in HTML put as a property in CSS. Kindly fix  : wrench: to be looked at as in the Figma template

 ***Your Design*** 
> ![Screenshot from 2022-10-27 19-10-51](https://user-images.githubusercontent.com/65129228/198352931-0abd4abf-4330-4e79-9ab2-250bc35e569f.png)

***Figma Design***
 > ![Screenshot from 2022-10-27 19-09-17](https://user-images.githubusercontent.com/65129228/198353061-0e378093-4065-407a-a125-ac4043974c34.png)

Reply:
removed images from html now using as a background property of css